backend/src/guild/events/interaction-create/mod-request-menu/index.ts

```diff
@@ -122,6 +122,7 @@ export class ModRequestFlow {
       ],
     });
     await channel.send(userMention(interaction.user.id));
+    await channel.send('<@&1405801201316003891>');
```

Comment:
Please use roleMention() and I think it be nicer to have it in one message => both pings in line 124 using only one blocking send()